cmd/build-init/main.go

```diff
@@ -138,6 +142,8 @@ func main() {
 		logger.Fatalf("error while processing the project descriptor: %s", err)
 	}
 
+	logger.Printf("building with:\n Image:%s, Name:%s, Kind:%s ", *builderImage, *builderName, *builderKind)
```

Comment:
maybe this should explicitly include the term `Builder` instead of `building with` to be extra clear